sympy/matrices/tests/test_matrices.py

```diff
@@ -17,6 +17,9 @@
 from sympy.utilities.pytest import raises, XFAIL, slow, skip
 from sympy.solvers import solve
 from sympy.assumptions import Q
+import warnings
+from sympy.utilities.exceptions import SymPyDeprecationWarning
+warnings.filterwarnings("ignore", category=SymPyDeprecationWarning)
```

Comment:
Ah.  I think if we want to test deprecated things, we should have a `test_deprecated.py` file that is separate.  That test can grab the warnings.

You can see `sympy/geometry/tests/test_point.py` line 66 for an example of a test that ensures a warning is raised.  Personally, I don't think testing a deprecation warning is very important.